integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiNegative.java

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
```

Comment:
Can we move this usecase to a existing MiiDomain class

Reply:
moved it to ItMiiDomain.java